podcastfy/core/audio.py

```diff
@@ -0,0 +1,106 @@
+import asyncio
+import atexit
+import os
+from concurrent.futures import ThreadPoolExecutor
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from typing import Optional, Dict, Union, List, cast, Tuple
+
+from pydub import AudioSegment
+
+from podcastfy.aiengines.tts.base import TTSBackend, SyncTTSBackend, AsyncTTSBackend
+from podcastfy.core.transcript import TranscriptSegment, Transcript
+
+
+class PodcastsAudioSegment:
+    """Represents an audio segment of the podcast."""
+
+    def __init__(self, filepath: Path, transcript_segment: Optional[TranscriptSegment] = None) -> None:
+        self.filepath = filepath
+        self.transcript_segment = transcript_segment
+        self._audio: Optional[AudioSegment] = None
+
+    @property
+    def audio(self) -> AudioSegment:
+        """Lazy-load the audio segment."""
+        if self._audio is None:
+            self._audio = AudioSegment.from_file(self.filepath)
+        return self._audio
+
+
+class AudioManager:
+    def __init__(self, tts_backends: Dict[str, TTSBackend], audio_format, n_jobs: int = 4, file_prefix: str = "", audio_temp_dir: str = None) -> None:
```

Comment:
default values should live in config.yaml

n_jobs: int = 4
file_prefix: str = "" 
audio_temp_dir: str = None